packages/taquito-rpc/src/rpc-client-modules/rpc-cache.ts

```diff
@@ -1165,4 +1170,58 @@ export class RpcClientCache implements RpcClientInterface {
       return response;
     }
   }
+
+  /**
+   *
+   * @param contract address of the contract we want to retrieve ticket balance of
+   * @param ticket object to specify ticketer, content type and content
+   * @param options contains generic configuration for rpc calls
+   * @description Access the contract's balance of ticket with specified ticketer, content type, and content.
+   * @example ticket{ ticketer: 'address', content_type: { prim: "string" }, content: { string: 'ticket1' } }
+   * @see https://tezos.gitlab.io/protocols/016_mumbai.html#rpc-changes
+   */
+  async getTicketBalance(
+    contract: string,
+    ticket: TicketTokenParams,
+    { block }: RPCOptions = defaultRPCOptions
+  ): Promise<string> {
+    const key = this.formatCacheKey(
+      this.rpcClient.getRpcUrl(),
+      RPCMethodName.GET_TICKET_BALANCE,
+      [block, contract],
+      ticket
+    );
+    if (this.has(key)) {
+      return this.get(key);
+    } else {
+      const response = this.rpcClient.getTicketBalance(contract, ticket, { block });
+      this.put(key, response);
+      return response;
+    }
+  }
+
+  /**
+   *
+   * @param contract address of the contract we want to retrieve ticket balance of
```

Comment:
it's probably a better description to say: '...retrieve all ticket balances from'